packages/flutter/lib/src/material/text_field.dart

```diff
@@ -163,6 +164,11 @@ class TextField extends StatefulWidget {
   /// Defaults to [TextAlign.start].
   final TextAlign textAlign;
 
+  /// The text direction for this TextField.
+  ///
+  /// Defaults to [TextDirection.ltr].
+  final TextDirection textDirection;
```

Comment:
The documentation should mention why you might want to override this instead of relying on `Directionality`.